lib/fathead/sass/parse_functions.py

```diff
@@ -0,0 +1,355 @@
+                anchor = self.parse_for_anchor(function_section[0])
+                example = self.parse_for_example(function_section[0])
+                parameter = self.parse_for_parameters(function_section[0])
+                abstract = "<p>" + description + "</p>" + method_signature
+                if example:
+                    abstract =  abstract + example
+                if parameter:
+                     abstract = "%s<br>%s"%(abstract, parameter)
+
+                url = self.create_url(anchor)
+                if function in functions:
+                    index = functions.index(function)
+                    data_elements = data[index]
+                    data_elements['abstract'] += abstract
+                else:
+                    functions.append(function)
+                    data_elements = {
+                        'function': function,
+                        'abstract': abstract,
+                        'url': url
+                    }
+                    data.append(data_elements)
+
+        self.parsed_data = data
+
+    def get_data(self):
+        """
+        Get the parsed data.
+        Returns:
+            self.parsed_data: Dict containing necessary data elements
+        """
+        return self.parsed_data
+
+
+class DataOutput(object):
+    """
+    Object responsible for outputting data into the output.txt file
+    """
+    def __init__(self, data):
+        self.data = data
+
+    def create_names_from_data(self, data_element):
+        """
+        Figure out the name of the function. Will contain the module name if one exists.
+        Args:
+            data_element: Incoming data dict
+
+        Returns:
+            Name, with whitespace stripped out
+
+        """
+        function = data_element.get('function')
+
+        dotted_name = '{}{}{}'.format(function, '.' if function  else '', function)
+        spaced_name = '{} {}'.format(function, function)
+
+        return dotted_name.strip(), spaced_name.strip()
+
+    def create_file(self):
+        """
+        Iterate through the data and create the needed output.txt file, appending to file as necessary.
+
+        """
+        with open('output.txt', 'a') as output_file:
+            for data_element in self.data:
+                if data_element.get('function'):
+                    name = data_element.get('function').encode('utf-8')
+                    abstract = data_element.get('abstract').encode('utf-8')
+                    url = data_element.get('url').encode('utf-8')
+                    
+                    list_of_data = [
+                        name,                       # unique name
+                        'A',                        # type is article
+                        '',                         # redirect data
+                        '',                         # ignore
+                        'sass functions,            # categories
```

Comment:
Looks like you have a syntax error here.

Please make sure you test you changes 👍